markhor_flippers/src/markhor_hw_interface_flippers.cpp

```diff
@@ -284,6 +288,11 @@ void MarkhorHWInterfaceFlippers::read()
   publishTarget();
   publishMotorCurrent();
   publishMotorBusVoltage();
+
+  joint_position_[0] = -front_left_drive_->GetSensorCollection().GetPulseWidthPosition() / (double)flipper_encoder_to_rad_coeff - 1.5707;
```

Comment:
the -1.5707 is equal to pi/2, or 90 degrees, this is due to a difference in referential between the urdf and the actual robot, we might change the urdf in the future to remove this, but to me it isn't critical at the moment as this would require changes to the controllers in the simulation

Reply:
But the urdf will be modified at some point in the future

Reply:
I agree since this whole file will most likely be changed entirely, I don't think we need to spend time cleaning it up